flowauth/backend/flowauth/admin.py

```diff
@@ -52,12 +52,22 @@ def list_all_capabilities():
     -----
     Responds with {<capability_name>: {"id":<capability_id>, "permissions":{<right>:False}}}
     """
+    default_permission_setting = current_app.config["DEMO_MODE"]
```

Comment:
One more observation: there is no indication of the _type_ here. One has to go to `flowauth/backend/config.py` to see that this is actually a boolean value - and thus be able to verify that it's ok to insert this into the permissions dict further below in the file.

I know it's more verbose, but how about
```
default_permission_setting = bool(current_app.config["DEMO_MODE"])
```
(or even be as explicit as
```
default_permission = True if (current_app.config["DEMO_MODE"] == True) else False
```
)?

I know it's redundant and less elegant and but it requires less cognitive overhead for the reader to verify the code. Just a suggestion though.